docs-ref-conceptual/get-started-tutorial-2-environment-syntax.md

```diff
@@ -0,0 +1,626 @@
+                          --output json
+```
+
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+# Variable block
+let "randomIdentifier=$RANDOM*$RANDOM"
+$location=eastus
+$resourceGroup="msdocs-tutorial-rg-00000000"
+$storageAccount="msdocssa$randomIdentifier"
+
+# Create a storage account.
+echo "Creating storage account $storageAccount in resource group $resourceGroup""
+az storage account create --name $storageAccount `
+                          --resource-group $resourceGroup `
+                          --location $location `
+                          --sku Standard_RAGRS `
+                          --kind StorageV2 `
+                          --output json
+```
+
+# [Cmd](#tab/cmd)
+
+```azurecli-interactive
+:: Variable block
+set randomIdentifier=%RANDOM%
+set resourceGroup="msdocs-tutorial-rg-00000000"
+set location=eastus
+set storageAccount="msdocssa%randomIdentifier%"
+
+:: Create a storage account.
+echo "Creating storage account %storageAccount% in resource group %resourceGroup%"
+az storage account create --name %storageAccount% ^
+                          --resource-group %resourceGroup% ^
+                          --location %location% ^
+                          --sku Standard_RAGRS ^
+                          --kind StorageV2 ^
+                          --output json
+```
+
+---
+
+The Azure CLI returns at least 100 lines of JSON as output when a new storage account is created. The following JSON dictionary output has fields omitted for brevity.
+
+```output
+{
+"accessTier": "Hot",
+"allowBlobPublicAccess": false,
+"creationTime": "yyyy-mm-ddT19:14:26.962501+00:00",
+"enableHttpsTrafficOnly": true,
+"id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/ msdocs-tutorial-rg-00000000/providers/Microsoft.Storage/storageAccounts/msdocssa00000000",
+"keyCreationTime": {
+  "key1": "yyyy-mm-ddT19:14:27.103127+00:00",
+  "key2": "yyyy-mm-ddT19:14:27.103127+00:00"
+},
+"kind": "StorageV2",
+"location": "eastus",
+"name": "msdocssa00000000",
+"primaryEndpoints": {
+  "blob": "https://msdocssa00000000.blob.core.windows.net/"
+},
+"primaryLocation": "eastus",
+"provisioningState": "Succeeded",
+"resourceGroup": "msdocs-tutorial-rg-00000000",
+"sku": {
+  "name": "Standard_RAGRS",
+  "tier": "Standard"
+},
+"statusOfPrimary": "available",
+"statusOfSecondary": "available",
+"tags": {},
+"type": "Microsoft.Storage/storageAccounts"
+}
+```
+
+## Create tags to practice quoting differences
+
+Using [az storage account update](/cli/azure/storage/account#az-storage-account-update), add tags to help you identify your storage account and learn about quoting differences. These examples demonstrate syntax differences for the following:
+
+* Values containing spaces
+* Quoting blank spaces
+* Escaping special characters
+* Using variables
+
+The `--tags` parameter accepts a space-separated list of key:value pairs.
+
+# [Bash](#tab/Bash)
+
+```azurecli-interactive
```

Comment:
I notice in the previous examples you are using the syntax with quotes, here you are using <>. Would it be helpful to keep the syntax consistent throughout the tutorial, or are you trying to show various forms of syntax for the user?